tests/robot-tests/tests/libs/no_javascript.py

```diff
@@ -0,0 +1,67 @@
+import requests
+from bs4 import BeautifulSoup
+
+def user_gets_parsed_html_from_page(url):
+    response = requests.get(url)
+    return BeautifulSoup(response.text, "html.parser")
+
+
+def user_checks_page_first_accordion_heading_is_not_a_button(parsed_page):
+    first_accordion = parsed_page.select_one("[data-testid='accordionSection']")
+    if not first_accordion:
+        raise AssertionError('No accordion section found')
+
+    heading = first_accordion.select_one('#publications-1-heading')
+    if not heading:
+        raise AssertionError('First accordion section heading wasn\'t found')
+    if heading.name == 'button':
+        raise AssertionError('First accordion section heading shouldn\'t be a button (with no javascript)')
+
+
+def user_checks_page_first_accordion_heading_does_not_have_attribute_aria_expanded(parsed_page):
+    first_accordion = parsed_page.select_one("[data-testid='accordionSection']")
+    if not first_accordion:
+        raise AssertionError('No accordion section found!')
+
+    heading = first_accordion.select_one('#publications-1-heading')
+    if not heading:
+        raise AssertionError('First accordion section heading wasn\'t found!')
+    if 'aria-expanded' in heading.attrs:
+        raise AssertionError('First accordion header contains "aria-expanded" attribute')
+
+
+def user_checks_page_first_accordion_section_contains_details(parsed_page):
```

Comment:
Rather than check for a details, I'd be tempted to check for text content (in the same way as the details).